src/pages/mainpage.module.css

```diff
@@ -0,0 +1,271 @@
+.wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  overflow-x: hidden;
+}
+
+.heroSection {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  min-height: calc(100vh - 89px);
+  background: linear-gradient(180deg, rgba(0, 0, 0, 0) 0%, rgba(27, 27, 27, 0.75) 76.18%, #1b1b1b 95.22%),
```

Comment:
```css  
  background-size: cover;
  background-position: center;
```
Using these attributes should help maintain the image's aspect ratio and achieve a natural effect.